dbms/src/test/java/org/polypheny/db/cypher/AggregateTest.java

```diff
@@ -144,14 +148,13 @@ public void singleMinMaxAggregateTest() {
 
 
     @Test
-    @Ignore// own min max impl necessary
```

Comment:
Can you enable this test?
However, something seems to be wrong with this class anyway, since this @Ignore is Junit 4... This should thrown an error.